### ISSUE TITLE
Spawn should facilitate inheritance (and thus evolution)

### ISSUE THREAD

**Issue by xparq** [xparq/sfml-test#41]
After #22, #32: Inherit some presumably interesting properties (like approx. color, moving patterns etc.) of the player body.

Also, a reminder for another issue, the inverse of #22 (add -> spawn): "remove_obj" should be upgraded to "Die".
(So, explicit object removal would become a low-level action.) Obj. lifespan should depend on various factors; initially just a random duration, but later interactions too etc.

This pair of basic lifecycle events should already establish an evolutionary sim. aspect/machinery.

Related ideas:
----------------
In multiuser -- OK, let's call it "multiplayer", assuming it's indeed becoming a game eventually... -- mode each player object would grow a family, different in some ways from the others'.

(NPC-like "emitters" should spawn families autonomously, too, esp. until networking.)

(Leave it open for both compet. and collab. interactions!)
